pkg/storage/endpoint/tso.go

```diff
@@ -64,7 +66,33 @@ func (se *StorageEndpoint) LoadTimestamp(prefix string) (time.Time, error) {
 }
 
 // SaveTimestamp saves the timestamp to the storage.
-func (se *StorageEndpoint) SaveTimestamp(key string, ts time.Time) error {
-	data := typeutil.Uint64ToBytes(uint64(ts.UnixNano()))
-	return se.Save(key, string(data))
+func (se *StorageEndpoint) SaveTimestamp(prefix string, key string, ts time.Time) error {
+	return se.RunInTxn(context.Background(), func(txn kv.Txn) error {
+		prefixEnd := clientv3.GetPrefixRangeEnd(prefix)
+		keys, values, err := txn.LoadRange(prefix, prefixEnd, 0)
```

Comment:
If my understanding is correct, for global location, we use '/pd/[cluster-id]' to do prefix scan, but there are a lot of things, under this path, irrelevant to tso. In order to "“LoadTimestamp will get all time windows of Local/Global TSOs from etcd and return the biggest one.”, shall we use the /pd/<cluster-id>/timestamp/<dc-X> for local TSOs?